src/GHCMod/Options/Commands.hs

```diff
@@ -60,6 +60,7 @@ data GhcModCommands =
   | CmdMapFile FilePath
   | CmdUnmapFile FilePath
   | CmdQuit
+  | CmdTest FilePath
```

Comment:
whoops that was just a quick and dirty merge to get it working since I'm already using this on a regular basis ;)